Protocol.md

```diff
@@ -377,18 +375,15 @@ Frame Contents
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     |                         Stream ID = 0                         |
     +-----------+-+-+---------------+-------------------------------+
-    |Frame Type |0|M|     Flags     |
+    |Frame Type |0|0|     Flags     |
     +-----------+-+-+---------------+-------------------------------+
     |0|                       Time-To-Live                          |
     +---------------------------------------------------------------+
     |0|                     Number of Requests                      |
     +---------------------------------------------------------------+
-                                Metadata
 ```
 
 * __Frame Type__: (6 bits = max value 63) 0x02 
-* __Flags__: (10 bits)
```

Comment:
Yes, there are use cases where we use metadata, c.f. my response to https://github.com/ReactiveSocket/reactivesocket/issues/201
We rely on that in our java implementation, I would like to keep it that way.